src/Symfony/Component/ErrorRenderer/ErrorRenderer/JsonErrorRenderer.php

```diff
@@ -40,12 +40,18 @@ public function render(FlattenException $exception): string
     {
         $debug = $this->debug && ($exception->getHeaders()['X-Debug'] ?? true);
 
+        if (404 === $exception->getStatusCode()) {
+            $message = 'Sorry, the page you are looking for could not be found.';
+        } else {
+            $message = $debug ? $exception->getMessage() : 'Whoops, looks like something went wrong.';
```

Comment:
it seems useless to me to add such hardcoded text to the error response. this does not provide any value to me. so I'm 👎 on this

Reply:
the other alternative would be not to show anything as now, so it would not provide any value either. I'm not sure what did you mean with :-1: here :/

My goal with this is to consistently have the `detail` message in the response on non-debug mode (currently it's missing).

Reply:
I disagree, it is not about useful or useless information to me, it's about a friendly message for better DX, have a consistent structure on debug/non-debug mode and also because (according to rfc) this "detail" member doesn't intend to give debugging information, that's my point.

Only `exception` should be part of the debugging information to me.